src/basic_recipes/title.jl

```diff
@@ -0,0 +1,30 @@
+@recipe(Title, titletext, plot) do scene
+    Theme(
+        textsize=40,
```

Comment:
Consider adding `default_theme(scene, Text) here to get the Text attributes as well, like colour.

Reply:
Thanks!